aten/src/ATen/native/native_functions.yaml

```diff
@@ -193,6 +193,10 @@
   dispatch:
     CompositeExplicitAutograd: _print
 
+- func: _sink_tokens(Tensor[] tokens) -> ()
+  dispatch:
+    CompositeExplicitAutograd: _sink_tokens
```

Comment:
I guess token type doesnt exist.. might be cleaner to move these op registrations to python, esp since they are temporary and dont get invoked at runtime except as meta. will let @zou3519 do full review here

Reply:
That is a good way of enforcing that these are compile-time tokens